tests/pflang-reg/pl-bug132-icmp6_or_portrange

```diff
@@ -0,0 +1 @@
+../../env pipe-lua-libpcap-match ../data/wingolog.pcap "icmp6 or portrange 14682-50101" 14951
```

Comment:
I see.  Perhaps then add

```sh
thisdir=$(dirname $0)
```

to the top of the scripts and use $thisdir as the base.  WDYT?